templates/cisco_ios_show_redundancy.textfsm

```diff
@@ -1,82 +1,91 @@
-Value SYSTEM_UPTIME_DAYS (\d+)
```

Comment:
Do you mind fixing the files that remove & replace? This makes it hard to see what actually changed and bound to miss something when reviewing.

Reply:
Added Fields as optional - System uptime years, System uptime weeks, Active uptime years, Active uptime weeks, standby uptime years, Standby uptime weeks
Updated peer failed status on line 73. my first push let me know if you have questions

Reply:
What is your OS you're editing in?

The line endings are what replacing all these files. If you're running linux, you can install dos2unix and run it against the files that are completely replaced rather than just showing your changes you made.

If you need help, I'm on our NTC public Slack as well.

Reply:
running on windows

Reply:
Yeah I think Windows is hijacking the line endings and causing this issue.

I will create a new PR that shows the actual changes and then tag you in it for these changes.